frontend/components/Memoriam.vue

```diff
@@ -238,25 +239,10 @@ export default {
 }
 
 .close {
-  display: flex;
-  justify-content: space-between;
-  align-items: center;
-  margin: 0 2rem;
+  margin: 0 1rem 1rem auto;
 
   @media (min-width: $tablet) {
-    align-items: flex-end;
-    margin: 0;
-  }
-
-  a {
-    margin-left: auto;
-  }
-
-  img {
-    width: 4rem;
-    height: 4rem;
-    border: 3px solid $surface;
-    border-radius: 50%;
+    margin: 0 0 1rem auto;
```

Comment:
These changes seem to have broken the share dropdown button when you've added a new memorial!
![Screenshot 2021-02-02 at 13 38 33](https://user-images.githubusercontent.com/76997670/106608184-fe6e4180-655b-11eb-962f-a8bc478818bf.png)